test/utils/fizzy_engine.cpp

```diff
@@ -46,13 +46,13 @@ bool FizzyEngine::parse(bytes_view input)
     return true;
```

Comment:
Can't comment there but why do we assign to `m_instance` in `parse` after this change?

Reply:
We don't any more.

Reply:
Yeah it was a subsequent commit.